src/libproc_macro/lib.rs

```diff
@@ -125,6 +125,10 @@ pub mod __internal {
         fn register_attr_proc_macro(&mut self,
                                     name: &str,
                                     expand: fn(TokenStream, TokenStream) -> TokenStream);
+
+        fn register_bang_proc_macro(&mut self,
```

Comment:
Okay. That doesn't affect anything in this PR, though, right? That's purely wording for documentation and stuff?